common/services/prismic/parsers.js

```diff
@@ -466,12 +467,12 @@ export function parseFormat(frag: Object): ?Format {
     : null;
 }
 
-function parseLink(url): ?string {
-  if (url) {
-    if (url.link_type === 'Web' || url.link_type === 'Media') {
-      return url.url;
-    } else if (url.link_type === 'Document' && isDocumentLink(url)) {
-      return `/${url.type}/${url.id}`;
+export function parseLink(link?: PrismicLink): ?string {
+  if (link) {
+    if (link.link_type === 'Web' || link.link_type === 'Media') {
```

Comment:
These are capitilised, where as the [type isn't](https://github.com/wellcomecollection/wellcomecollection.org/pull/5636/files#diff-5fea9633ec4028dfe4db8897f45e92ea8cdc3d4cba04dc4b06b8c67b180c210dR20)? Which is right?

Reply:
You're right – it _should_ be capitalised (that's what comes back from Prismic). The confusion arises because to set these, the Prismic config values are lowercase. Thanks.